android/modules/ui/src/java/ti/modules/titanium/ui/PickerRowProxy.java

```diff
@@ -29,6 +30,18 @@ public PickerRowProxy()
 	}
 
 	@Kroll.getProperty @Kroll.method
+	public String getColor()
+	{
+		return (String)getProperty(TiC.PROPERTY_COLOR);
+	}
+
+	@Kroll.setProperty @Kroll.method
+	public void setColor(String value)
```

Comment:
Could use `color` instead of `value`. Maybe `value` was used in `setTitle` to remove ambiguity as `title` already exists as field? Your call. 

Otherwise, looks good to merge!